### PR TITLE
fix redefinition warning for RESET_COLOR

### PR DESCRIPTION
**Pull request by Dark-Passenger**

### CODE REVIEW

File: include/fmt/format-inl.h [hunk 1]
```diff
@@ -86,8 +86,8 @@ inline int fmt_snprintf(char *buffer, size_t size, const char *format, ...) {
 # define FMT_SWPRINTF swprintf
 #endif // defined(_WIN32) && defined(__MINGW32__) && !defined(__NO_ISOCEXT)
 
-const char RESET_COLOR[] = "\x1b[0m";
-const wchar_t WRESET_COLOR[] = L"\x1b[0m"; 
+static FMT_CONSTEXPR_DECL const char RESET_COLOR[] = "\x1b[0m";
+static FMT_CONSTEXPR_DECL const wchar_t WRESET_COLOR[] = L"\x1b[0m";
```
Comment (vitaut) [inline include/fmt/format-inl.h]: Why static?

**Comment by vitaut** [comment]
Merged with some tweaks in https://github.com/fmtlib/fmt/commit/ec218a3ad16feb9eb69a3d467b054cfbb7f47f4d. Thanks!

File: include/fmt/format-inl.h (hunk 1 above)
Comment (Dark-Passenger) [reply include/fmt/format-inl.h]: that would be copy-pasta mistake on my part.

**Comment by Dark-Passenger** [comment]
awesome sorry about the late reply and some pretty cool tweaks if I may say so myself.

**Pull request closed without merging**